content/en/docs/setup/install/multicluster/simplified/index.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Simplified Multicluster Install (experimental)
+description: Configure an Istio mesh spanning multiple Kubernetes clusters.
+weight: 1
+keywords: [kubernetes,multicluster]
+---
+
+This guide describes how to configure an Istio mesh that includes multiple Kubernetes clusters using a simplified experimental approach.
+We hope to continue developing this functionality in coming releases, so we'd love your feedback on the overall flow.
+
+We focus here on the details of getting a multicluster mesh wired up, refer to [multicluster deployment model](/docs/setup/deployment-models/#multiple-clusters) for
+additional background information. We'll show how to connect two clusters that are on the same network together, along
+with a third cluster that's on a different network.
+
+Using the approach shown in this guide results in an instance of the Istio control plane being deployed in every cluster
+within the mesh. Although this is a common configuration, other more complex topologies are possible, but have to be done
+using a more manual process, not described herein.
+
+## Before you begin
+
+The procedures we describe here are primarily intended to be used with relatively pristine clusters,
+where Istio hasn't already been deployed. We hope to expand support in the future to existing cluaters.
+
+For the sake of explanation, this guide assumes you have created three Kubernetes clusters:
+
+- A cluster named `cluster-east-1` on the network named `network-east`.
+- A cluster named `cluster-east-2` on the network named `network-east`.
+- A cluster named `cluster-west-1` on the network named `network-west`.
+
+These clusters shouldn't have Istio on them yet. Take a look at the [platform setup instructions](/docs/setup/platform-setup)
+for any special instructions for your particular environment.
```

Comment:
Do you need to say something about the first two needing to have network connectivity?

Reply:
Done.